pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -635,6 +636,13 @@ public void start() throws PulsarServerException {
                         + "authenticationEnabled=true when authorization is enabled with authorizationEnabled=true.");
             }
 
+            if (config.isLoadBalancerEnabled() && LinuxInfoUtils.isLinux()) {
```

Comment:
If I set `loadBalancerOverrideBrokerNicSpeedGbps` I should not get the error, right?